### DISCUSSION TITLE
AWS SecretsManager

### DISCUSSION THREAD

**Discussion by darrylkuhn** [drakkan/sftpgo#1422] [Q&A]
I'd like to configure the data provider (database username/password) through AWS SecretsManager. I see that the SecretsManager is listed as a dependency in the [go.mod](https://github.com/drakkan/sftpgo/blob/main/go.mod), however in looking through [the docs](https://github.com/drakkan/sftpgo/tree/main/docs) (which are great by the way) I didn't see any mention of using the SecretsManager to configure the data provider. Is that not a current feature or am I not connecting the dots. Thanks!

**Comment by drakkan** [comment]
This is not supported sorry, the SecretManager is used only for the AWS container offering

**Reply by darrylkuhn** [reply]
Got it - appreciate the response!
